### ISSUE TITLE
Qt 6: github.com crashes when logged in

### ISSUE THREAD

**Issue by twouters** [qutebrowser/qutebrowser#7422]
Qutebrowser crashes after login on github when running in temp basedir:

```
$ qutebrowser -T -d -R github.com
```

I don't have a coredump yet because I don't have debug symbols available right now.

```
$ qutebrowser --version
         ______     ,,
    ,.-"`      | ,-` |
  .^           ||    |
 /    ,-*^|    ||    |
;    /    |    ||    ;-*```^*.
;   ;     |    |;,-*`         \
|   |     |  ,-*`    ,-"""\    \
|    \   ,-"`    ,-^`|     \    |
 \    `^^    ,-;|    |     ;    |
  *;     ,-*`  ||    |     /   ;;
    `^^`` |    ||    |   ,^    /
          |    ||    `^^`    ,^
          |  _,"|        _,-"
          -*`   ****"""``

qutebrowser v2.5.2
Git commit: 5e11e6c7d on makepkg (2022-09-16 15:00:19 +0200)
Backend: QtWebEngine 6.4, based on Chromium 102.0.5005.177 (from api)
Qt: 6.4.0 (compiled 6.3.2)

CPython: 3.10.7
PyQt: 6.3.1

sip: no
colorama: 0.4.5
jinja2: 3.1.2
pygments: 2.13.0
yaml: 6.0
adblock: 0.6.0
objc: no
PyQt6.QtWebEngineCore: 6.3.1
pdf.js: 2.16.105 (/usr/share/pdf.js/build/pdf.js)
sqlite: 3.39.4
QtNetwork SSL: OpenSSL 1.1.1q  5 Jul 2022

Style: QFusionStyle
Platform plugin: xcb
OpenGL: Intel, 4.6 (Compatibility Profile) Mesa 22.1.7
Platform: Linux-5.19.12-arch1-1-x86_64-with-glibc2.36, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.10/site-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib/qt6, data path: /usr/share/qt6

Paths:
cache: /home/twouters/.cache/qutebrowser
config: /home/twouters/.config/qutebrowser
data: /home/twouters/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser

Autoconfig loaded: yes
Config.py: no config.py was loaded
Uptime: 0:00:00
```

**Comment by The-Compiler** [comment]
I can reproduce, it seems to be a regression in QtWebEngine 6.4 sadly. No debug symbols ready at the moment either, though.

**Comment by The-Compiler** [comment]
Already reported and fixed upstream:

- [[QTBUG-107144] WebEngine crashes on github.com - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-107144)
- [Do not intercept websocket connection when there is no associated frame (If042e415) · Gerrit Code Review](https://codereview.qt-project.org/c/qt/qtwebengine/+/436316)

I've asked the Archlinux maintainer to backport that patch, so I'd expect it to be fixed with qt6-webengine 6.4.0-2 there, once that's out.

**Comment by bodograumann** [comment]
I can confirm that qt6-webengine 6.4.0-2 fixes the issue on Arch Linux.
Thanks for the quick reaction!
